testsuite/MDAnalysisTests/topology/test_fetch_pdb.py

```diff
@@ -0,0 +1,84 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the Lesser GNU Public Licence, v2.1 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+# Note need to make test not run when there is no internet
+
+from requests.exceptions import HTTPError
```

Comment:
I don't understand why we need to install and import `requests` just for the exception. I think you should be using [urllib.error.HTTPError](https://docs.python.org/3/library/urllib.error.html#urllib.error.HTTPError) instead.
```suggestion
from urllib.error import HTTPError
```

Reply:
`requests` is a required dependency for [pooch](https://www.fatiando.org/pooch/latest/install.html), and by default it throws this exception when the HTTP request doesn't work out.

Reply:
I guess the alternative is like monkey patching so the two exception share the same reference? But that sounds so needlessly unnecessary since `requests` has to be installed as a dependency for `pooch` to work. I move the import inside the `has_pooch()` function for clarity.

Reply:
I think you solved the problem by importing it if pooch is available; see my comments.